src/components/logger/history.ts

```diff
@@ -98,6 +98,33 @@ export async function openLogfile() {
   }
 }
 
+export async function deleteFromProbeRequests(limit: number) {
+  const getIdsToBeDeleted = `SELECT id FROM probe_requests order by created_at asc limit ${limit}`
```

Comment:
so if the limit is 1, which is the default, this will only delete 1 request? is that enough to keep the db size under the max_db_size?

Reply:
done, by keep deleting untill file size is reached.